s27125-HubertJozwiak/zajecia2/zad_dom.txt

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+if [ $# -ne 1 ]; then
+    echo "Nieprawidłowa liczba argumentów"
+    exit 1
+fi
+
+if [ ! -f $1 ]; then
+    echo "Argument musi być plikiem"
+    exit 1
+fi
+
+plik=$(cat $1)

```

Comment:
I w tym miejscu jesteś pewien, że skoro sprawdziłeś tylko, że plik istnieje to już automatycznie możesz go odczytać tak?